chapters/flow/index.md

```diff
@@ -1,5 +1,8 @@
-# Using Flow API to Compose Your Jina Workflow
+## Using Flow API to Compose Your Jina Workflow
 
+##### Summary: In this section, you will get to know how to construct the flow via different approaches. 
+
+### Feature description and expected outcome
 In search systems, tasks such as indexing often involve multiple steps: preprocessing, encoding, storing, etc. In Jina's architecture, each step is implemented by an Executor and wrapped by a Pod. This microservice design makes the whole pipeline flexible and scalable. Accomplishing a task is then orchestrating all these Pods to work together, either sequentially or in parallel; locally or remotely. 
```

Comment:
Could we replace orchestrating with linking?

Reply:
Sure! Could you elaborate more or do we have examples on that? 😂